src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -30,7 +30,7 @@ import { Observable } from 'rxjs';
 @Component({
   selector: 'app-entry-box',
   templateUrl: './entry-box.component.html',
-  styleUrls: ['./entry-box.component.scss'],
+  styleUrls: ['./entry-box.component.scss', '../../../shared/styles/dashboard-boxes.scss'],
```

Comment:
Assuming that you removed the empty entry-box.component.scss, remove it from this line too